.evergreen/config.yml

```diff
@@ -203,6 +203,19 @@ functions:
         content_type: application/x-gzip
         display_name: "fuzz.tgz"
 
+  upload-raw-test-suite:
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: src/go.mongodb.org/mongo-driver/test.suite
+        optional: true
+        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${version_id}/${build_id}/logs/${task_id}-${execution}-test.suite.txt
```

Comment:
Tar'ing the test.suite.txt file before PUT would result in a faster upload.

Reply:
Done